lib/Agrammon/Outputs/FilterGroupCollection.pm6

```diff
@@ -79,7 +79,18 @@ class Agrammon::Outputs::FilterGroupCollection {
             @results
         }
         else {
-            [%!values-by-filter.map({ .key.filters => .value })]
+#            @results = [%!values-by-filter.map({ .key.filters => .value })]
+            my @results;
+            for $!provenance.keys.sort(*.module.load-order) -> $filter-set {
```

Comment:
You can't rely on having `$!provenance` set always; if think you'll have to test for it and if it's missing, just do what we originally did, and only if its there then include it.